server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -312,7 +312,7 @@ public class IndicesService extends AbstractLifecycleComponent
      */
     public static final Setting<PathType> CLUSTER_REMOTE_STORE_PATH_PREFIX_TYPE_SETTING = new Setting<>(
         "cluster.remote_store.index.path.prefix.type",
-        PathType.FIXED.toString(),
+        PathType.HASHED_PREFIX.toString(),
```

Comment:
Will this break things while upgrading? Do different values of this setting assume remote store to be configured in certain way?

Reply:
Yes, this makes sense. I will undo this change.